### ISSUE TITLE
About the numbering after ~

### ISSUE THREAD

**Issue by xiki-tempula** [selimsami/qforce#16]
I don't think one needs to do something about this and the end result would always be the same but I wonder if you felt that it is better to have a global QM method number?

I have parameterised a batch of molecules yesterday with method A and then with method B, the fragments are then named as ~1 for method A and ~2 for method B.

Today I have parameterised another batch of molecules but with method B first and then with method A.
I noticed that ~1 is used to name method B and ~2 is used to name method A. For some molecules that share some fragments from the batch from yesterday, I got ~1 and ~2 in the same folder denoting the same method. Though it won't affect the results, I still think it is a bit confusing.

**Comment by selimsami** [comment]
I agree about the confusing. It used to be that the file names contained the method, basis set, dispersion, etc. But the file names then quickly became unruly. 

Perhaps method could also be included in the hash but I don't love the idea. Do you have any suggestions?

**Comment by xiki-tempula** [comment]
I think it might be good to have a global identifier (at least from my naive understanding of how the module works).

So instead of having ~/qforce_fragments/CO_H8C7O1_fe01b2e37d8e82d910aede5f1cf7f09b/qm_method_1

You would have ~/qforce_fragments/qm_method_1

Then the ~N would be corresponding to the ~/qforce_fragments/qm_method_N and everything else could be the same.

I think intuitively, people would be cool of opening ~/qforce_fragments/CO_H8C7O1_fe01b2e37d8e82d910aede5f1cf7f09b and finding that everything starts from 10 and the suffix would be corresponding to a defined QM method.

than

open ~/qforce_fragments/CO_H8C7O1_fe01b2e37d8e82d910aede5f1cf7f09b and finds everything starts with 1 but the suffix from different folder coorepodning to a different QM method.

**Comment by selimsami** [comment]
That makes sense, good idea.

However, currently ~N does a bit more than just separate methods though (i.e., you could end up with a different N using the same method on different molecules with same fragments). But that is something I intend to change in the future. When I do that, I will think of implementing your suggestion as well.

**Comment by xiki-tempula** [comment]
I see.

`(i.e., you could end up with a different N using the same method on different molecules with same fragments)`

I thought that the N is only used to differentiate between different QM methods so I got a bit confused when I see different N when using the same QM method but I'm looking forward to the update.
